sale_cancel_reason/wizard/cancel_reason.py

```diff
@@ -0,0 +1,27 @@
+# © 2013 Guewen Baconnier, Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo import fields, models
+
+
+class SaleOrderCancel(models.TransientModel):
+
+    """ Ask a reason for the sale order cancellation."""
+
+    _name = "sale.order.cancel"
```

Comment:
Name already in use by base Odoo

Reply:
If you change, don't forget migration script.